components/match2/wikis/halo/match_group_input_custom.lua

```diff
@@ -60,21 +61,17 @@ function CustomMatchGroupInput.processMap(_, map)
 	return map
 end
 
--- called from Module:Match/Subobjects
```

Comment:
Was this comment here just no longer accurate? Because it wasn't actually called from there.

Reply:
It was called from Match/Subobjects, and now it's no longer called from Match/Subobjects.

Reply:
It wasn't called from there though. Current `main` branch code has no reference to it. Neither does your PR remove such a reference

Reply:
It's called from here: https://github.com/Liquipedia/Lua-Modules/blob/main/components/match2/commons/match_subobjects.lua#L31

`wikiSpec.processOpponent(frame, args)` calls into the wiki specific module which the calls into `CustomMatchGroupInput.processOpponent`

Reply:
Alright 👍